README.md

```diff
@@ -14,9 +14,33 @@ Deploying Haskell code onto [AWS Lambda] using [Serverless].
 * [Stack]
 * [NPM]
 * [Docker]
+* [Serverless]
```

Comment:
This can be installed via NPM - the above section should probably be "Prerequisites" to be clearer.